symbols/arm9.yml

```diff
@@ -1235,6 +1246,17 @@ arm9:
         
         r0: item ID
         return: bool
+    - name: IsChest
```

Comment:
Can we use the term "treasure box" to match the category name, rather than "chest"? It's less ambiguous than "chest" which could conceivably also refer to things like the Miracle/Wonder chest.

Reply:
Sure